configs/erfnet/erfnet.yml

```diff
@@ -1,5 +1,5 @@
 Collections:
-- Name: erfnet
```

Comment:
based on my limited knowledge, erfnet might be an algorithm not backbone

Reply:
This PR currently delete `Collections` dict in some backbone readme such as HRNet, Swin Transformer and so on because they may be supported in MMClassification. Other methods which is algorithm rather than backbones all keep  `Collections` dict but the `Collection['Name']` and `Models['In Collection']` are the same value.